apps/server/src/schema/user/invite-operations.ts

```diff
@@ -214,23 +233,35 @@ builder.mutationFields((t) => ({
             frontEndInvitedUserUrl.search = searchParams.toString();
             logger.info(`Confirm invited non-existing user ${email}: ${frontEndInvitedUserUrl.toString()}`);
 
-            const newUser = await createInvitedUser(email, emailType, args.role, ctx.organizationId);
-            await Promise.all([
-              sendOrganizationInviteConfirmEmail({
-                firstName: newUser.firstName,
-                lastName: newUser.lastName,
-                email,
-                organizationName: organization.name,
-                expirationInDays: invitationExpirationInDays,
-                actionUrl: frontEndInvitedUserUrl.toString(),
-              }),
-              setConfirmInvitedUserRedis(inviteHash, newUser.id, ctx.organizationId),
-            ]);
+            const newUser: AError | User = await createInvitedUser(email, emailType, args.role, ctx.organizationId);
+            if (!(newUser instanceof AError)) {
+              await Promise.all([
+                sendOrganizationInviteConfirmEmail({
+                  firstName: newUser.firstName,
+                  lastName: newUser.lastName,
+                  email,
+                  organizationName: organization.name,
+                  expirationInDays: invitationExpirationInDays,
+                  actionUrl: frontEndInvitedUserUrl.toString(),
+                }),
+                setConfirmInvitedUserRedis(inviteHash, newUser.id, ctx.organizationId),
+              ]);
+            } else {
+              return new AError('Error creating invited user');
```

Comment:
```suggestion
              return { message: newUser.message, email };
```
This is what the errors expect